pkg/pod/v1alpha1/kubeclient.go

```diff
@@ -0,0 +1,96 @@
+package v1alpha1
```

Comment:
Please keep this at
`pkg/kubernetes/pod/v1alpha1/kubernetes.go`

Please raise another PR to rename `kubernetes.go` to `kubeclient.go`

Reply:
changed